test/RLP.js

```diff
@@ -0,0 +1,74 @@
+const RLP = artifacts.require("./RLPMock.sol"),
+    ethUtil = require("ethereumjs-util"),
+    Utils = require('./lib/utils.js');
+
+
+contract('RLP library', function (accounts) {
+    let rlpMock;
+    before(async () => {
+        rlpMock = await RLP.new();
+
+    });
+    let dataArray = ['2', '5', '6'];
+    describe('Test Cases ', async () => {
+
+        it('should pass when RLP data is correct', async () => {
+
+            dataInNibbles = ethUtil.rlp.encode(dataArray).toString('hex')
+                , result = await rlpMock.toRLPItem.call('0x' + dataInNibbles)
+                , lengthInBytes = dataInNibbles.length;
+            assert.equal((lengthInBytes / 2), result[0].toString(10));
+            assert.isAtLeast(result[1].toString(10), 0);
+
+        })
+
+        it('should fail to return memory location when RLP is incorrect or length is 0', async () => {
+            let result = await rlpMock.toRLPItem.call("");
+            assert.equal(result[1].toString(10), 0);
+            assert.equal(result[0].toString(10), 0);
+
+        })
+
+        it('should fail when RLP is incorrect with option strict as true', async () => {
+
+            await Utils.expectThrow(rlpMock.toRLPItemStrict.call(dataArray, true));
+
+        })
+
+        it('should pass when RLP list is correct', async () => {
```

Comment:
Please test the fail case(s), when the provided data is not a list. Can `ret` be `false` in `ret := lt(byte(0, mload(memPtr)), 0xC0)` with a length > 0?

Please also revise the description of this test similarly to description revisions requested above.